nnoir-onnx/nnoir_onnx/operators/max_pool.py

```diff
@@ -26,12 +26,12 @@ def __init__(self, node, *args):
                 self.pads = attr.ints
 
             # opset version >= 10
-            elif attr.name == 'ceil_mode':
-                raise UnsupportedONNXOperation(self.node, 'attribute `ceil_mode` is not supported')
+            elif attr.name == 'ceil_mode' and attr.i != 0:
+                raise UnsupportedONNXOperation(self.node, 'only value 0 is supported for attribute `ceil_mode`')
 
             # opset version >= 11
             elif attr.name == 'dilations':
-                raise UnsupportedONNXOperation(self.node, 'attribute `dialations` is not supported')
+                raise UnsupportedONNXOperation(self.node, 'attribute `dilations` is not supported')
             else:
                 raise UnsupportedONNXOperation(self.node, 'unknown attribute {}'.format(attr.name))
```

Comment:
`unknown attribute ceil_mode` is raised when `ceil_mode=0`

Do you intend to do like this ?
```python
            # opset version >= 10
            elif attr.name == 'ceil_mode':
                if attr.i != 0:
                    raise UnsupportedONNXOperation(self.node, 'only value 0 is supported for attribute `ceil_mode`')
```

And, plz. add a test to `test_max_pooling.py` for case `ceil_mode=0`.

Reply:
Good catch, I will fix this.